1.1/spec/18-Bibliography.adoc

```diff
@@ -1,72 +1,48 @@
 [bibliography]
```

Comment:
Does OGC doc builder support SpecRef? That might obviate the need to create yetanother reference list.

Reply:
I don't think so: we are dealing with ASCIIDOC but I will check and see if it can handle BibText etc.

I am aligning the _Bibliography_ and _Normative References_ sections now.